src/relations/mysql_provider.py

```diff
@@ -309,7 +317,38 @@ def _on_database_broken(self, event: RelationBrokenEvent) -> None:
         relation_id = event.relation.id
         try:
             self.charm._mysql.delete_users_for_relation(relation_id)
-            logger.info(f"Removed user for relation {relation_id}")
+            logger.info(f"Removed user(s) for relation {relation_id}")
         except MySQLDeleteUsersForRelationError:
-            logger.error(f"Failed to delete user for relation {relation_id}")
+            logger.error(f"Failed to delete user(s) for relation {relation_id}")
+            return
```

Comment:
is this return necessary? isnt going to return since it's the end of the method?